.github/workflows/version-branch.yml

```diff
@@ -0,0 +1,48 @@
+name: Replicate Commits to Version Branch
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  replicate:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+
+    - name: Get version
+      id: get_version
+      run: |
+        VERSION=$(python -c '
+        from openwisp_utils import VERSION
+        print(f"{VERSION[0]}.{VERSION[1]}.x")
```

Comment:
Wouldn't this produce something like `1.2.x`? We are using branch names without the x now.